src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -55,6 +57,9 @@ class ModuleExecutor(
                     )
                 }
 
+                IssueUpdateContextCache.forEach(::applyUpdatesAndTransitions)
+                IssueUpdateContextCache.clear()
```

Comment:
This is currently in the executeModules method, which means that in case one function or one module does something wrong, the entire run will fail and last-run will remain at the old time.

This should move into each module's execute function and instead return a failed module response for that module, so that we only save issue keys where modules actually failed and don't have to run through all issues if something bad happened.

Reply:
Then that will not group these API requests if I understand you correctly

Reply:
That's exactly what happens when one module fails.
If you can adapt that code to collect failed tickets here, that would be fine as well.